wazo_dird/plugins/default_json/http.py

```diff
@@ -77,6 +78,57 @@ def get(self, profile):
         return response
 
 
+class LookupByUUID(LegacyAuthResource, DisplayAwareResource):
```

Comment:
You should use `AuthResource` instead of `LegacyAuthResource`
The only difference is that `AuthResource` will return errors in the "new" format and legacy is used for the old API to avoid bumping the version